web/tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "lib": ["es2016", "es2017", "dom"],
     "skipLibCheck": true,
     "esModuleInterop": true,
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "resolveJsonModule": true
   },
   "include": ["./src/**/*"],
   "exclude": ["./src/__tests__/*"]
```

Comment:
Can we do without these config changes now? Implicit any in TypeScript is not the best rule to have disabled.